### ISSUE TITLE
Selectable "Administration" columns in dashboard

### ISSUE THREAD

**Issue by laclaro** [privacyidea/privacyidea#2470]
Administrators might want to configure which columns from the audit log they see on the dashboard.

We would need
1. a policy to define which the columns to display
2. some code from the audit log html for the formatting

**Comment by plettich** [comment]
Is this fixed with #2500 ?

**Comment by laclaro** [comment]
> Is this fixed with #2500 ?

No, the columns are statically listed. If you hide the column, it will be actually throw an error in the dashboard since the accessed field is not in `auditentry`.
